data/json/requirements/melee.json

```
@@ -0,0 +1,682 @@
+        [ "lc_kukri", -1 ],
+        [ "mc_kukri", -1 ],
+        [ "ch_kukri", -1 ],
+        [ "hc_kukri", -1 ],
+        [ "qt_kukri", -1 ],
+        [ "tanto", -1 ],
+        [ "karambit", -1 ],
+        [ "kirpan", -1 ],
+        [ "butterfly_swords", -1 ]
+      ]
+    ]
+  },
+  {
+    "id": "basic_thrusting_swords",
+    "type": "requirement",
+    "//": "Any weapon that you could thrust with and stab someone.",
+    "tools": [
+      [
+        [ "sword_metal_fencing", -1 ],
+        [ "estoc_fake", -1 ],
+        [ "estoc_inferior", -1 ],
+        [ "fencing_foil", -1 ],
+        [ "fencing_foil_sharpened", -1 ],
+        [ "shock_foil", -1 ],
+        [ "fencing_epee", -1 ],
+        [ "fencing_epee_sharpened", -1 ],
+        [ "shock_epee", -1 ],
+        [ "fencing_sabre", -1 ],
+        [ "fencing_sabre_sharpened", -1 ],
+        [ "shock_sabre", -1 ],
+        [ "rapier_fake", -1 ],
+        [ "sharpened_pipe", -1 ]
+      ]
+    ]
+  },
+  {
+    "id": "real_thrusting_swords",
+    "type": "requirement",
+    "//": "Any real, quality weapon designed to be used in a thrusting fashion.  Needs to be a sword, not a spear.  This includes fencing weaponry.",
+    "tools": [
+      [
+        [ "estoc", -1 ],
+        [ "lc_estoc", -1 ],
+        [ "mc_estoc", -1 ],
+        [ "ch_estoc", -1 ],
+        [ "hc_estoc", -1 ],
+        [ "qt_estoc", -1 ],
+        [ "sword_cane", -1 ],
+        [ "rapier", -1 ],
+        [ "lc_rapier", -1 ],
+        [ "hc_rapier", -1 ],
+        [ "ch_rapier", -1 ],
+        [ "mc_rapier", -1 ],
+        [ "qt_rapier", -1 ],
+        [ "lc_talwar", -1 ],
+        [ "mc_talwar", -1 ],
+        [ "hc_talwar", -1 ],
+        [ "ch_talwar", -1 ],
+        [ "qt_talwar", -1 ],
+        [ "cavalry_sabre", -1 ],
+        [ "lc_cavalry_sabre", -1 ],
+        [ "mc_cavalry_sabre", -1 ],
+        [ "hc_cavalry_sabre", -1 ],
+        [ "ch_cavalry_sabre", -1 ],
+        [ "qt_cavalry_sabre", -1 ],
+        [ "jian", -1 ],
+        [ "lc_jian", -1 ],
+        [ "mc_jian", -1 ],
+        [ "hc_jian", -1 ],
+        [ "ch_jian", -1 ],
+        [ "qt_jian", -1 ],
+        [ "broadsword", -1 ],
+        [ "lc_broadsword", -1 ],
+        [ "mc_broadsword", -1 ],
+        [ "ch_broadsword", -1 ],
+        [ "hc_broadsword", -1 ],
+        [ "qt_broadsword", -1 ]
+      ]
+    ]
+  },
+  {
+    "id": "basic_short_swords",
+    "type": "requirement",
+    "//": "Any basic thing you could use as a short sword, exlcudes high-quality things.",
+    "tools": [ [ [ "null", -1 ] ] ]
+  },
+  {
+    "id": "real_short_swords",
+    "type": "requirement",
+    "//": "Any real, edged short swords you could practice with, like broadswords.  I reckon that anything with a straight, sharp point to it that primarily swings side-to-side can qualify.",
+    "tools": [
+      [
+        [ "khopesh", -1 ],
+        [ "sword_xiphos", -1 ],
+        [ "sword_bronze", -1 ],
+        [ "sword_bayonet", -1 ],
+        [ "bio_blade_weapon", -1 ]
```

```suggestion
        [ "survivor_machete", -1 ],
        [ "survivor_machete_qt", -1 ]
```
The combat machetes are more appropriate than the bionic sword, the bionic blade is not considered a proper sword for the martial arts because of its very short length.